tools/js-sdk-release-tools/src/common/changlog/automaticGenerateChangeLogAndBumpVersion.ts

```diff
@@ -24,16 +24,22 @@ import { getApiVersionType } from '../../xlc/apiVersion/apiVersionTypeExtractor.
 import { fixChangelogFormat, getApiReviewPath, getNpmPackageName, getSDKType, tryReadNpmPackageChangelog } from '../utils.js';
 import { tryGetNpmView } from '../npmUtils.js';
 
-export async function generateChangelogAndBumpVersion(packageFolderPath: string) {
+export async function generateChangelogAndBumpVersion(packageFolderPath: string,  options?: { apiVersion: string, sdkReleaseType: string }) {
```

Comment:
I remember it will be used by cli, please also add options to the cli to support new options